resources.cpp

```diff
@@ -19,22 +19,7 @@ Bitmap *load_bitmap(BitmapName name) {
 	BITMAP bitmap;
 	GetObject(bitmap_resource, sizeof(BITMAP), &bitmap);
 
-	GLubyte *data = new GLubyte[BITMAP_WH * BITMAP_WH];
-
-
-	for (size_t y = 0; y < BITMAP_WH; y++) {
-		for (size_t x = 0; x < BITMAP_WH; x++) {
-			int i_source = y * BITMAP_WH + x;
-			int i_target = (BITMAP_WH - y - 1) + BITMAP_WH * x;
-	
-			data[i_target] = ((GLubyte *) bitmap.bmBits)[i_source];
-		}
-	}	 
-	// ------------------------------------------------------------
-	//                      GET ROTATED
-	//                         Idiot
```

Comment:
It turns out that, this whole time, I was actually drawing the sprites _sideways_ - which was the only reason I needed to rotate the bitmap here. I fix that in this PR so this rotating is no longer necessary.

Reply:
Huh, I was always wondering what you're rotating.